hkl/configuration.py

```diff
@@ -245,8 +245,7 @@ class DCSample:
     (the reciprocal of the crystal) in an orthonormal system.
     """
 
-    # TODO: Once py38 is dropped, re-enable the default value setting
-    U: List[List[float]]  # = field(default_factory=list[list[float]])
+    U: List[List[float]]
```

Comment:
It wasn't clear whether you had already tried initializing the default value like this?
```suggestion
    U: List[List[float]] = field(default_factory=lambda: [[]])
```

Reply:
I'll try that next.

Reply:
Will also apply to `UB` (above).